app/components/Task.tsx

```diff
@@ -137,14 +145,14 @@ const Task = () => {
                   <div>
                     {item.tasks.length > 0 ? (
```

Comment:
a shorter version would look like this

```
{item.tasks.length > 0 && (
  <div key={index} className='flex flex-col'>
    {item.tasks[0]}
  </div>
)}
```

Reply:
im not sure about this one. i need it to print out 'no task' if there is no task written.

Reply:
oh i see. resolved for now